tools/test/examples/examples_lib.py

```diff
@@ -405,5 +415,10 @@ def update_mbedos_version(config, tag, examples):
             update_dir =  basename(repo_info['repo']) + "/mbed-os"
             print("\nChanging dir to %s\n" % update_dir)
             os.chdir(update_dir)
-            subprocess.call(["mbed-cli", "update", tag, "--clean"])
+            result = subprocess.call(["mbed-cli", "update", tag, "--clean"])
             os.chdir("../..")
+            if result:
+                return result:
```

Comment:
@OPpuolitaival found this, but the correct thing here would be:

```suggestion
                return result
```

Reply:
Yup copy paste there me thinks.